app/src/test/java/org/mozilla/fenix/ext/ImageButtonTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.ext
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import org.mozilla.fenix.TestApplication
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import io.mockk.mockk
+import io.mockk.verify
+import android.widget.ImageButton
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+
+class ImageButtonTest {
+    val mockIb: ImageButton = mockk(relaxed = true)
+
+    @Test
+    fun `Hide and disable`() {
+        mockIb.hideAndDisable()
+        verify { mockIb.setVisibility(4) }
+        verify { mockIb.setEnabled(false) }
```

Comment:
Call verification usually indicates the test is coupled to the implementation and is a red flag (but sometimes necessary). In this case, this test would pass if this was the method implementation despite there being a bug:
```kotlin
fun hideAndDisable() {
    this.visibility = View.INVISIBLE
    this.isEnabled = false
    this.visibility = View.VISIBLE
}
```

A better practice is to test return values or the final state of an object, for which the test will stay the same even if the implementation changes. In this case, you should be able to inflate an ImageButton via robolectric (though I don't know how hard that is in practice) and verify the final state after the test. Here and in the other test.